snap/Dockerfile.aarch64

```diff
@@ -0,0 +1,15 @@
+FROM arm64v8/ubuntu:xenial
+RUN apt-get update && \
+  apt-get dist-upgrade --yes && \
+  apt-get install --yes \
+  snapcraft curl file sudo jq squashfs-tools && \
```

Comment:
This one doesn't install `make`, but the others do. Is that intentional?

Reply:
oversigh